rust/core-lib/src/editor.rs

```diff
@@ -73,6 +73,7 @@ pub struct Editor<W: Write> {
     scroll_to: Option<usize>,
 
     style_spans: Spans<Style>,
+    soft_spans: Spans<()>,
```

Comment:
While currently empty, `()` could be replaced with a bool or enum once weak spans are implemented